pkg/scheduler/framework/session_plugins.go

```diff
@@ -341,6 +345,25 @@ func (ssn *Session) PredicateFn(task *pod_info.PodInfo, job *podgroup_info.PodGr
 	return nil
 }
 
+func (ssn *Session) NodePartitionFn(job *podgroup_info.PodGroupInfo, tasks []*pod_info.PodInfo, nodes api.NodePartition) ([]api.NodePartition, error) {
+	var err error
+	partitions := []api.NodePartition{nodes}
```

Comment:
This always adds a partition of all the nodes, maybe it should be empty and some default plugin will fill it up? This way we can also remove it as an option if needed.